templates/taxi/index.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+
+{% block content %}
+<ul>
+    <li>Number of cars: {{num_drivers}}</li>
+    <li>Number of drivers: {{num_manufacturers}}</li>
+    <li>Number of manufacturers: {{num_cars}}</li>
```

Comment:
There is a mismatch between the variable names used here and those provided in the context from the view. The correct variables should be `num_cars`, `num_drivers`, and `num_manufacturers` as per the context dictionary in the view function. Please update the template to use the correct variable names.